docs/src/development/local/docksal.md

```diff
@@ -1,128 +1,200 @@
+To get your project, pull it with the following command:
 
 ```bash
-fin run-cli 'platform get <PROJECT_ID> -e <BRANCH_NAME> <PROJECT_DIRECTORY>'
+fin pull init --hosting-platform='platformsh' --hosting-site=<PROJECT_ID> --hosting-env=<ENVIRONMENT_NAME> <TARGET_DIRECTORY_NAME>
 ```
 
-If you already have Platform.sh CLI installed locally, you can use that instead.
+This creates a directory with the specified name with all your files and code.
+It also adds a `.docksal` directory with all necessary Docksal configuration.
+These files are ignored by Platform.sh.
+
+## 5. Add commands
+
+Docksal doesn't automatically copy over any commands you have in your [build flavor](../../administration/../configuration/app/app-reference.md#build)
+and [hooks](../../configuration/app/hooks/_index.md).
+To get you project running like on Platform.sh, you have to add the commands to Docksal.
+
+The `.docksal/commands` directory should already have one command (`init`) like the following:
+
+```bash {location=".docksal/commands/init"}
+#!/usr/bin/env bash
+
+fin project rm -f
+fin project start
 
-```bash
-platform get <PROJECT_ID> -e <BRANCH_NAME> <PROJECT_DIRECTORY>
 ```
 
-## Initializing a Platform.sh project
+This command removes any running Docksal containers and starts new ones.
+You can add new commands here that run on `fin init`
+or create a new file to run on `fin <NAME_OF_FILE>`.
 
-To start a new Docksal project, initialize the configuration with the `fin config generate` command and specify the `docroot` flag.
+You can add your build flavor and hook commands after these commands.
+That way the Docksal CLI container is running and can process commands.
 
-```bash
-fin config generate --docroot=web
+For example, to install dependencies, use a command like the following depending on your dependency manager:
+
+{{< codetabs >}}
+
+---
+title=npm
+file=none
+highlight=false
+---
+
+```bash {location=".docksal/commands/init"}
+#!/usr/bin/env bash
+
+fin project rm -f
 fin project start
+
+cd "$PROJECT_ROOT"
+echo "Installing dependencies..."
+npm install
+
 ```
+<--->
 
-The `web` directory is one of the many different items that can be set for the document.
-If this is different or changes over time running the following will fix this.
+---
+title=Composer
+file=none
+highlight=false
+---
+
+```bash {location=".docksal/commands/init"}
+#!/usr/bin/env bash
+
+fin project rm -f
+fin project start
+
+cd "$PROJECT_ROOT"
+echo "Installing dependencies..."
+fin composer install
 
-```bash
-fin config set docroot=XXX # Replacing XXX with the new document root.
 ```
 
-## Customizing a Platform.sh project
+{{< /codetabs >}}
+
+## 6. Run your project
+
+Now your project is ready to run.
+In your project directory, run `fin init`.
+
+The command returns the project URL `http://<DIRECTORY_NAME>.docksal/`.
+Open this URL to see your project running.
+
+## What's next
+
+You've got a project running on a local web server.
+Now you can add customizations.
```

Comment:
💅 An example could be to add `mkcert` to have https on docksal.
https://docs.docksal.io/tools/mkcert/#body-inner

Reply:
The examples were intended to be the following sections.